code/modules/clothing/under/miscellaneous.dm

```diff
@@ -530,13 +530,65 @@
 	worn_state = "mai_yang"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS
 
+/obj/item/clothing/under/cheongsamwhite

```

Comment:
Once again, might be worth parenting.

```
/obj/item/clothing/under/cheongsam
	name = "white cheongsam"
	desc = "A Chinese dress that hugs the body. This one is white, embroidered with a bright golden dragon."
	icon = 'icons/obj/clothing/cheongsams.dmi'
	icon_state = "cheongsamwhite"
	item_state = "cheongsamwhite"
	worn_state = "cheongsamwhite"
	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS
	contained_sprite = 1

/obj/item/clothing/under/cheongsam/red
	name = "red cheongsam"
	desc = "A Chinese dress that hugs the body. This one is red, with a golden dragon design that climbs the entire garment."
	icon_state = "cheongsamred"
	item_state = "cheongsamred"
	worn_state = "cheongsamred"
```

Would make spawning and filtering them easier. Also remove some duplicate var defs.